VTFLib/VTFFile.cpp

```diff
@@ -2444,7 +2446,7 @@ vlBool CVTFFile::GenerateMipmaps(vlUInt uiFace, vlUInt uiFrame, VTFMipmapFilter
 	}
 
 	bool bOk = true;
-	for (vlUInt32 i = 1; i <= GetMipmapCount(); ++i)
+	for (vlUInt32 i = 1; i < GetMipmapCount(); ++i)
```

Comment:
Good catch

Reply:
Actually, I think I'm a bit confused here- Is the mip count at *minimum* 1? Or can it be 0 when there are no mips, but base image data?

Reply:
Checking other vtfs, the mip count is the number of 'mips' including the original image.

Reply:
Gotcha, thanks for clarifying!